core/templates/dev/head/pages/notifications_dashboard/notifications_dashboard.html

```diff
@@ -33,39 +33,43 @@ <h2 class="oppia-page-heading">Notifications</h2>
 
       <div ng-if="recentNotifications.length > 0">
         <md-card ng-repeat="notification in recentNotifications" class="oppia-dashboard-row"
-          ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername}" ng-click="navigateToItem(notification.activity_id, notification.type)">
-          <div class="oppia-notification-content">
-            <span class="oppia-notification-type">
-              <span ng-if="notification.type === 'feedback_thread'"
-                    title="Feedback thread">
-                <i class="material-icons">&#xE0B9;</i>
-              </span>
-              <span ng-if="notification.type === 'exploration_commit'"
-                    title="Exploration commit">
-                <i class="material-icons">&#xE150;</i>
-              </span>
-            </span>
-            <span class="oppia-notification-author-username">
-              <strong>
-                <a ng-if="notification.author_username" ng-click="navigateToProfile($event, notification.author_username)"><[notification.author_username]></a>
-                <span ng-if="!notification.author_username">
-                  Anonymous
+                 ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername}"
+                 style="position: relative;">
```

Comment:
@rackstar17 I remember having checked that there only here is the class `oppia-dashboard-row` applied, and no place else. If that is still the case, then it makes sense to move this to css instead of inline styles!